source/_reference/basics.coffee

```diff
@@ -481,6 +488,12 @@ module.exports = exports =
                      The resource can't be found.
                      """
       ,
+        key: "too-many-results"
+        http: "413"
+        description: """
+                     When using the `events.get` method in batch calls or websockets that generate too many results. For calls yielding large results, call the API method directly, narrow the request scope or page the request.
+                     """
```

Comment:
I changed it a bit differently: "The `events.get` **method in batch or websocket call** yielded too many results. Call the API method directly, narrow the request scope or page the request."

This error can only happen batch and websockets calls.